src/main/java/de/muenchen/allg/itd51/wollmux/event/handlers/OnProcessTextDocument.java

```diff
@@ -3,8 +3,9 @@
 import org.slf4j.Logger;
```

Comment:
Kannst du die noch in das Package ...document.commands verschieben. Ich denke dort passt sie fachlich am besten hin.

Reply:
oder nur in document, jenachdem was dir besser gefällt